tensorflow/compiler/tf2tensorrt/convert/ops/binary_ops.cc

```diff
@@ -57,6 +67,22 @@ class ConvertBinaryImpl {
           "' received both input as constant");
     }
 
+   if (!not_supported_ops.empty() && params.use_implicit_batch) {
+      const auto& end = not_supported_ops.end();
+      if (std::find(not_supported_ops.begin(), end, op) != end) {
+        return errors::Unimplemented(
+            "Binary op: '", op, "' is not supported in implicit batch mode");
+      }
+    }
+
+    if (both_tensors) {
+      // Need to check if both inputs should be tensors.
```

Comment:
This comment is the same as the line before, it is better without this comment.

Reply:
Done